etl/db.py

```diff
@@ -173,3 +173,90 @@ def get_all_datasets(archived: bool = True, db_conn: Optional[MySQLdb.Connection
 
 def dict_to_object(d):
     return type("DynamicObject", (object,), d)()
+
+
+def get_info_for_etl_datasets(db_conn: Optional[MySQLdb.Connection] = None) -> pd.DataFrame:
+    if db_conn is None:
+        db_conn = get_connection()
+
+    # First, increase the GROUP_CONCAT limit, to avoid the list of chart ids to be truncated.
+    GROUP_CONCAT_MAX_LEN = 4096
+    cursor = db_conn.cursor()
+    cursor.execute(f"SET SESSION group_concat_max_len = {GROUP_CONCAT_MAX_LEN};")
+    db_conn.commit()
+
+    query = """\
+    SELECT
+        q1.datasetId AS dataset_id,
+        d.name AS dataset_name,
+        q1.etlPath AS etl_path,
+        d.isArchived AS is_archived,
+        d.isPrivate AS is_private,
+        q2.chartIds AS chart_ids
+    FROM
+        (SELECT
```

Comment:
Looks like having `catalogPath` in `datasets` table would be useful. I'll add it one day, I promise...